.github/workflows/publish.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     permissions:
       contents: none
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@924441154cf3053034c6513d5e06c69d262fb9a6  # v1.13.0
-    if: (github.event_name == 'push' || github.event_name == 'pull_request')
+    if: (github.event_name == 'push' || (github.event_name != 'workflow_dispatch' && github.event_name != 'schedule' && github.event_name != 'labeled'))
```

Comment:
This makes me question myself. You said `(github.event_name == 'push' || github.event_name == 'pull_request')` was picking up labeling event? If a labeling event really emits `github.event_name == 'labeled'`, it should not have been possible.

Reply:
All I know was that when I relabeled a backport PR (removing and then adding `skip-changelog-checks` to get that one to run again), the wheel building also started again...

Reply:
Okay this is not working. I just applied build all wheels label and this job ran anyway. So I probably need to try things out on a fork when I have time to smoke out the actual event context. Turning this into draft for now.